{"status":"FAIL","msg":"FAIL:\n Uncaught: { message: \"Expected a [object Test262Error] but got a TypeError\" }\n    at ../../test262/harness/assert.js:95\n    at anonymous (../../test262/test/built-ins/Temporal/PlainYearMonth/get-prototype-from-constructor-throws.js:38)\n\n","path":"../../test262/test/built-ins/Temporal/PlainYearMonth/get-prototype-from-constructor-throws.js"}